Add Discord to contact informations (#23789)
mdn/content: files/en-us/mdn/community/pull_requests/index.md

We want MDN Web Docs to be a welcoming, friendly community that we can all be proud of.
All participants must follow our [Code of Conduct](https://github.com/mdn/content/blob/main/CODE_OF_CONDUCT.md) which means adhering to [Mozilla's Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
Be polite and constructive when opening pull requests, writing review comments, interacting with the pull request author or other community members.

> If anyone has engaged in behavior that is potentially illegal or makes you or someone else feels unsafe, unwelcome, or uncomfortable, you are encouraged to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).

Author: > I think the original is correct here :)

My bad didn't read the full sentence :( Rephrasing it to make it direct.
```suggestion
If you or someone else has experienced behavior that is potentially illegal or makes you feel unsafe, unwelcome, or uncomfortable, we encourage you to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).
```

> If anyone has engaged in behavior that is potentially illegal or makes you or someone else feels unsafe, unwelcome, or uncomfortable, you are encouraged to [report it](https://www.mozilla.org/en-US/about/governance/policies/participation/reporting/).

Reviewer: @akasyaagroup.com
